src/test/java/uk/gov/hmcts/ccd/data/caseaccess/AMCaseUserRepositoryComponentTest.java

```diff
@@ -0,0 +1,133 @@
+package uk.gov.hmcts.ccd.data.caseaccess;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.ccd.BaseTest;
+import uk.gov.hmcts.ccd.data.helper.AccessManagementQueryHelper;
+import uk.gov.hmcts.reform.amlib.DefaultRoleSetupImportService;
+import uk.gov.hmcts.reform.amlib.models.ResourceDefinition;
+
+import javax.sql.DataSource;
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.collection.IsIterableContainingInAnyOrder.containsInAnyOrder;
+import static uk.gov.hmcts.reform.amlib.enums.AccessType.ROLE_BASED;
+import static uk.gov.hmcts.reform.amlib.enums.RoleType.IDAM;
+import static uk.gov.hmcts.reform.amlib.enums.SecurityClassification.PUBLIC;
+
+@Transactional
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = {AccessManagementQueryHelper.class})
+@ActiveProfiles("test")
+public class AMCaseUserRepositoryComponentTest extends BaseTest {
```

Comment:
Why change of name to add `Component`?

Reply:
Reverted the name to AMCaseUserRepositoryTest.java.